lib/uaa/cli/curl.rb

```diff
@@ -27,15 +27,16 @@ class CurlCli < CommonCli
     define_option :data, "-d", "--data <data>", "data included in request body"
     define_option :header, "-H", "--header <header>", "header to be included in the request"
     define_option :insecure, "-k", "--insecure", "makes request without verifying SSL certificates"
+    define_option :quiet, "-q", "--quiet", "Quiet, show body only"
```

Comment:
I am not sure if "quiet" is the best choice of the word for this option.  It reminds me options that are used to hide verbose messages and such.  How about "-b" and "--bodyonly" instead?
Additionally, how about changing the description to to clarify the option is for response?  Something like "show body only in response"?